packages/server/lib/plugins/index.js

```diff
@@ -113,7 +113,7 @@ const init = (config, options) => {
       version: pkg.version,
     })
 
-    ipc.send('load', config)
+    ipc.send('load', { ...config, testingType: options.testingType })
```

Comment:
This can go with the other options added to config above.

```
    _.extend(config, {
      projectRoot: options.projectRoot,
      configFile: options.configFile,
      version: pkg.version,
      testingType: options.testType,
    })
    
    ipc.send('load', config)
```